### DISCUSSION TITLE
USB support

### DISCUSSION THREAD

**Discussion by some103** [86Box/86Box#4680] [Ideas]
Some 90s motherboards had USB/headers for it, my idea is to be able to attach a device connected to the host computer, like VirtualBox allows to do.

**Comment by Yoghoo** [comment]
Saw that this was requested a couple of times in the past. But it was rejected unfortunately as it probably needs some kind of guest side application. See for example here: https://github.com/86Box/86Box/discussions/2907. 

Would really like this as well. Mainly for USB floppy and CD-ROM support to get an even better retro machine feel when using 86Box. :)

**Reply by jelabarre59** [reply]
It would also be useful as an alternative to having to run VirtualBox/VMWare/QEMU to run a USB scanner under Win2K/WinXP (a Scanjet 7400 with transparency adapter).  It's inconsistent and unreliable under VBox (I need to set it up on a MSWin machine, but I'm using a Linux system to create the VM)
